### PR TITLE
Eq 1506 census content tweaks

### PR DESCRIPTION
**Pull request by shahi645**
### What is the context of this PR?
Various census content tweaks, details found here:
https://trello.com/c/rcAGdAWB/1506-content-tweaks

### How to review 
Open `census_household` and verify that content tweaks match that on trello card

### CODE REVIEW

File: data/en/census_household.json [hunk 1]
```diff
@@ -1481,7 +1464,7 @@
                                             "value": "Republic of Ireland"
                                         },
                                         {
-                                            "label": "Elsewhere (please specify)",
+                                            "label": "Please specify current name of country",
```
Comment (dwyeradam) [inline data/en/census_household.json]: I think this was meant to replace the  "label": "Enter current name of country" in "id": "country-of-birth-england-answer-other" and "id": "country-of-birth-wales-answer-other". Then just remove (please specify) from the existing label.

**Review by dwyeradam — CHANGES_REQUESTED** [review]

**Review by dwyeradam — CHANGES_REQUESTED** [review]
You missed the Welsh equivalent: _country-of-birth-wales-question._

File: data/en/census_household.json [hunk 2]
```diff
@@ -3681,15 +3566,7 @@
                     "description": "",
                     "questions": [{
                         "title": "How do you usually travel to work?",
-                        "guidance": {
-                            "content": [{
-                                    "description": "Select the option for the longest part, in distance, of your journey"
-                                },
```
Comment (charlotte-hirst) [inline data/en/census_household.json]: should there be a line break between these two lines, i dont think there should be as they are related

File: data/en/census_household.json [hunk 3]
```diff
@@ -324,12 +324,12 @@
...
-                                    "People who are temporarily outside the UK for <b>less than 12 months</b>",
-                                    "People staying temporarily who usually live in the UK but do not have another UK address, for example relatives, friends",
-                                    "Other people who usually live here, including anyone temporarily away from home"
+                                    "people who usually live outside the UK who are staying in the UK for <b>three months or more</b>",
+                                    "people who work away from home within the UK  if this is their permanent or family home",
+                                    "members of the Armed Forces if this is their permanent or family home",
+                                    "people who are temporarily outside the UK for <b>less than 12 months</b>",
+                                    "people staying temporarily who usually live in the UK but do not have another UK address, for example relatives, friends",
+                                    "pther people who usually live here, including anyone temporarily away from home"
```
Comment (charlotte-hirst) [inline data/en/census_household.json]: other not pther at start of line

File: data/en/census_household.json [hunk 4]
```diff
@@ -421,12 +421,12 @@
...
-                                    "Same-sex civil partner is used to describe the relationship between two people who have legally registered a civil partnership.",
-                                    "If none of the options fully reflects your relationship, please select the one which you feel best describes your situation."
+                                    "select the relationships between members of your household",
+                                    "if you are the parents of adopted children, select the ‘Mother or father’ option to show your relationship to them.",
+                                    "if you have foster children living with you, select the ‘Unrelated’ option.",
+                                    "include half-brothers and half-sisters in the ‘Step-brother or step-sister’ category.",
+                                    "same-sex civil partner is used to describe the relationship between two people who have legally registered a civil partnership.",
+                                    "if none of the options fully reflects your relationship, please select the one which you feel best describes your situation."
```
Comment (charlotte-hirst) [inline data/en/census_household.json]: these shouldnt be lower case as not an include list but to be honest it needs re-writing but could we move to guidance under question not in panel please

**Review by charlotte-hirst — CHANGES_REQUESTED** [review]

File: data/en/census_household.json (hunk 3 above)
Comment (shahi645) [reply data/en/census_household.json]: my bad, will change!

File: data/en/census_household.json (hunk 2 above)
Comment (dwyeradam) [reply data/en/census_household.json]: @charlotte-hirst This isn't in keeping with our patterns. We're essentially asking for guidance without the guidance box. We shouldn't be using breakpoints in the description so I think we need to discuss what is needed here.

**Review by dwyeradam — APPROVED** [review]

**Review by charlotte-hirst — APPROVED** [review]
LGTM

**Comment by jonnyshaw89** [comment]
@shahi645 On you!

**Pull request merged**